### PR TITLE
ENH: methods for loading COVID data

### PR DESCRIPTION
**Pull request by aflaxman**

### CODE REVIEW

File: src/vivarium_uw_covid/data/loader.py [hunk 1]
```diff
@@ -101,7 +101,103 @@ def _load_em_from_meid(location, meid, measure):
     return utilities.sort_hierarchical_data(data)
 
 
-# TODO - add project-specific data functions here
+seiir_dir = '/ihme/covid-19/seir-pipeline-outputs'
```
Comment (rmudambi) [inline src/vivarium_uw_covid/data/loader.py:104]: this should be in `paths.py`

**Review by rmudambi — COMMENTED** [review]
I'm not sure where you are loading this data to. Generally this `loader.py` is used in conjunction with `builder.py` to build an Artifact. Is that goal here?

**Comment by aflaxman** [comment]
ev

> 
> 
> I'm not sure where you are loading this data to. Generally this `loader.py` is used in conjunction with `builder.py` to build an Artifact. Is that goal here?

eventually I want to build an artifact for this, but for now, I'm loading it directly into memory.

File: src/vivarium_uw_covid/data/loader.py (hunk 1 above)
Comment (aflaxman) [reply src/vivarium_uw_covid/data/loader.py:104]: helpful, will move

**Pull request merged**